src/amo/pages/Home/index.js

```diff
@@ -341,24 +348,29 @@ export class HomeBase extends React.Component {
           />
         )}
 
-        {renderFeaturedCollection(1)}
-
         <LandingAddonsCard
           addonInstallSource={INSTALL_SOURCE_FEATURED}
-          addons={shelves.popularExtensions}
-          className="Home-PopularExtensions"
-          header={i18n.gettext('Popular extensions')}
-          footerText={i18n.gettext('See more popular extensions')}
+          addons={shelves.popularAddons}
+          className="Home-PopularAddons"
+          header={i18n.gettext('Popular themes')}
+          footerText={i18n.gettext('See more popular themes')}
           footerLink={{
             pathname: '/search/',
             query: {
-              addonType: ADDON_TYPE_EXTENSION,
+              q: 'themes',
```

Comment:
Do we really want/need `q=themes` here? I know that's in the requirements doc, but I don't understand it. Normally just having the `addonType` is enough. @devaneymoz, can you comment on this?

Reply:
@bobsilverberg Sorry but again I'm unclear where precisely we're looking at on the page?

Reply:
@devaneymoz It's the link in the header of the shelf where we list popular themes:

![screenshot 2019-01-16 15 43 26](https://user-images.githubusercontent.com/142755/51277452-b5171d80-19a5-11e9-8cf3-ae8db5bb3428.png)

Reply:
> Do we really want/need `q=themes` here?

I don't know.. That was in the Google doc, but maybe not. I removed it for now, we can still add it later.